README.md

```diff
@@ -20,7 +20,6 @@
 # Apache OpenWhisk Project Release Management
 
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
-[![Build Status](https://travis-ci.com/apache/openwhisk-release.svg?branch=master)](https://travis-ci.com/github/apache/openwhisk-release)
```

Comment:
Could we add the status badge via Github workflow too or is this removed for some reason?
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
Once the workflow is running, I plan to add the badge in a follow up PR.